### ISSUE TITLE
Debloater made "System" in Windows Settings Unusable

### ISSUE THREAD

**Issue by Limeay** [Sycnex/Windows10Debloater#279]
I can't even get to Windows settings -> System portion for the display settings such as GPU Scheduling Hardware or VRR, it simply closes out and it also messed up my start menu tiles to where it won't show up my pinned applications.

Nevermind on the Start Menu Tiles, they work again after a reboot. BUT PLEASE fix the Windows Settings "System" whenever i press it, it just closes it out and gives me a crash log in %temp%

**Comment by manishchoudharylive** [comment]
Yes, I'm having the same issue. Can't open my settings app anymore, n to make it work, would have to reinstall all the bloatware.
Please fix this. 
Ps. When I press Win+I, it flashes and closes again.
Can anyone suggest the version where this perticular issue wasn't present.

**Comment by Sycnex** [comment]
What version are you guys using of Windows?

**Comment by Linkkie** [comment]
> What version are you guys using of Windows?

Version 2004, I have the same exact problem. Trying to revert but no succes. Need to change the hardware scheduling under display settings.

**Comment by Limeay** [comment]
Yep, Exactly. v2004 and i can't get this thing to work. I've tried everything.

**Comment by Sycnex** [comment]
Unfortunately you may need to do a Reset of Windows. I am working on a new version of the script that will allow you to choose specifically what is removed and such. 

I will also add in a feature to create a System Restore point prior to changes being made.

**Comment by Limeay** [comment]
I made a comment on this post.
Pretty much summing up I think it can be fixed by simply installing the next latest windows 10 version... v2009 or 2010 whenever it comes out, because I'm sure I seen this issue before and after installing new windows it fixed it.
https://github.com/Sycnex/Windows10Debloater/issues/277

**Comment by manishchoudharylive** [comment]
Hey, update here from my earlier [comment](https://github.com/Sycnex/Windows10Debloater/issues/279#issuecomment-674365147).
Windows Settings app is opening again, after a windows update.
It required restarting the laptop. After it all went back to normal -> (as in with all the bloatware).
I'll wait for a proper fix to this problem though, before using it again.

**Comment by Limeay** [comment]
Uh... my display portion of windows settings app is still not working and my mouse is killing me because of the latency issue from the broken windows app... I've tried everything online and dism and no luck. I've installed the newest windows update, no fix.

**Comment by Limeay** [comment]
For anyone that want to have their windows 10 immersivecontrolpanel fixed, the solution is either to reformat your OS or to upgrade to the latest Windows 10 version, for ex: this issue messed up my OS in August, Right now it's October 2020, the newest version just came out. Updated to it and now i don't have this issue anymore and my pc isn't so laggy after it broke my OS.

**Comment by deeharrybalsagna** [comment]
Why isn’t this on the front page… I just used this yesterday and almost everything is corrupted and failing. I spent 90% of my free time trying to stop the bleeding.
